src/bundle/Resources/views/themes/admin/content_type/part/field_definition_form.html.twig

```diff
@@ -0,0 +1,56 @@
+{% trans_default_domain 'content_type' %}
+
+{% form_theme form '@ezdesign/ui/form_fields.html.twig' %}
+
+{% set value = form.vars.value %}
+{% set name = value.names[language_code] ?? value.names[content_type.mainLanguageCode] ?? ''  %}
```

Comment:
this line seems strange, maybe it would be better to put in in two lines, instead of double shorthand?